requirements/env_unidist.yml

```
@@ -51,4 +51,4 @@ dependencies:
       - flake8-no-implicit-concat
       - flake8-print
       # The `numpydoc` version should match the version installed in the `lint-pydocstyle` job of the CI.
-      - numpydoc==1.1.0
+      - numpydoc==1.1.0
```

This should help revert the change:
```suggestion
      - numpydoc==1.1.0

```